tree/tree/src/TEntryList.cxx

```diff
@@ -12,38 +12,9 @@
 /** \class TEntryList
 \ingroup tree
 
-A List of entry numbers in a TTree or TChain.
-
-There are two types of entry lists:
+\legacy{TEventList}
```

Comment:
```suggestion
\legacy{TEntryList}
```

Reply:
`TEntryList` is the (more) modern interface and should not be marked legacy for now.